internal/model/loadbalancer.go

```diff
@@ -20,12 +20,34 @@ import (
 	"github.com/golang-jwt/jwt"
 )
 
+type AllowedIPBlock struct {
+	Cidr   string   `json:"cidr"`
+	Except []string `json:"except"`
+}
+
+type PolicyPort struct {
+	Protocol corev1.Protocol `json:"protocol"`
+	Port     *int32          `json:"port,omitempty"`
+	EndPort  *int32          `json:"end-port,omitempty"`
+}
+
+type NetworkPolicy struct {
+	Name            string           `json:"name"`
+	AllowedIPBlocks []AllowedIPBlock `json:"allowed-ip-blocks"`
+	Ports           []PolicyPort     `json:"ports"`
+}
+
+type PolicyAssignment struct {
+	Address         string   `json:"address"`
+	NetworkPolicies []string `json:"network-policies"`
+}
+
 type PortForward struct {
 	Protocol             corev1.Protocol `json:"protocol"`
 	InboundPort          int32           `json:"inbound-port"`
 	DestinationAddresses []string        `json:"destination-addresses"`
 	DestinationPort      int32           `json:"destination-port"`
-	Policy               string          `json:"policy"`
+	Policy               string          `json:"policy"` // TODO: <== What does this do
```

Comment:
I think this was intended to be used for the balancing policy, which currently only supports and defaults to round-robin.

Reply:
Can I rename it to "BalancePolicy" or "LoadBalancePolicy" to avoid confusion with the NetworkPolicies?

Reply:
... though that would warrant a bump to API v2. So not worth it I guess.

Reply:
I renamed the internal (and unused) field Policy to BalancePolicy but left the json field alone.